src/hclust.jl

```diff
@@ -662,9 +751,17 @@ Returns the dendrogram as a [`Hclust`](@ref) object.
  - `uplo::Symbol` (optional): specifies whether the upper (`:U`) or the
    lower (`:L`) triangle of `d` should be used to get the distances.
    If not specified, the method expects `d` to be symmetric.
+- `branchorder::Symbol` (optional): algorithm to determine order leaves and branches.
```

Comment:
In my initial suggestion it was
```suggestion
- `branchorder::Symbol` (optional): algorithm to order leaves and branches.
```

Reply:
Ah sorry, github interface does this thing when I try to select that makes it hard to see. Copy-paste fail there

<img width="605" alt="Screen Shot 2019-06-21 at 8 40 00 AM" src="https://user-images.githubusercontent.com/3502975/59922993-32850500-9400-11e9-971e-bb888f4ceb29.png">